paper/paper.md

```diff
@@ -64,9 +64,11 @@ The Julia package [``CapacityExpansion``](https://github.com/YoungFaithful/Capac
 ``TimeSeriesClustering`` is the first package to provide broadly applicable unsupervised learning methods specifically for time series in Julia [@Bezanson:2017].
 There are several other related packages that provide useful tools for these tasks, both in Julia and in the general open-source community, and we describe them in order to provide guidance on the broader tools available for these kinds of modeling problems.
 
-The [``Clustering``](https://github.com/JuliaStats/Clustering.jl) in Julia package provides a broad range of clustering methods and and allows computation of clustering validation measures. ``TimeSeriesClustering`` provides a simplified workflow for clustering time series, and works on top of the ``Clustering`` package by making use of a subset of the clustering methods implemented in the ``Clustering`` package.
+The [``Clustering``](https://github.com/JuliaStats/Clustering.jl) in package in Julia provides a broad range of clustering methods and and allows computation of clustering validation measures. ``TimeSeriesClustering`` provides a simplified workflow for clustering time series, and works on top of the ``Clustering`` package by making use of a subset of the clustering methods implemented in the ``Clustering`` package.
```

Comment:
The [``Clustering``](https://github.com/JuliaStats/Clustering.jl) package in Julia ?